WebAPI/Api/SomeServiceInterface.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Epam\WebAPI\Api;
+
+interface SomeServiceInterface
+{
+
+    /**
+     * @param string $test
+     * @return mixed
```

Comment:
Please, do not use mixed return type, specify it

Reply:
thank you, fixed